### PR TITLE
Project Proposal added

### PR DESCRIPTION
**Pull request by alvaxgln**

### CODE REVIEW

File: projects/alvaxgln/proposal.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Milestones could be:
+* Implementing basic edge operations, such as edge-flip, edge collapse, edge split
+* Implementing angle based smoothing
+* Algorithm for basic edge swaps
+* Algorithm for advanced edge classification and handling
+
+Problems could arise with the angle based smoothing, since that might be difficult to implement, and also changes Vertex positions.
+A different issue might be that since in the original paper the meshes were created in the remehsing scheme the author can make certain guarantees about them, that might not be true for all meshes.
+
```
Comment (JohannesMerkt) [inline projects/alvaxgln/proposal.md:39]: Maybe turning a non-manifold mesh into a manifold mesh is needed too

**Review by MarlQ — COMMENTED** [review]
It would be interesting to see what kind of artifacts the algorithm manages to remove/reduce. 
How do you want to design the controls?

**Comment by nico778** [comment]
Question related to Marcel's comment. Should the algorithm even reduce artifacts? Does your project only try to regularize the mesh connectivity while avoiding to change the mesh(vertex positions) itself, or does it try to regularize the mesh connectivity and therefore also the mesh?

File: projects/alvaxgln/proposal.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Then Use edge operations to regularize Vertices
+* Use regular subdivision to get a high regular mesh
+
+(A Problem of that approach would be loss of details during simplification)
+
+## References
+
+
+- Ligang Liu, Lecture slides on Delaunay Triangulation and Mesh Generation, http://staff.ustc.edu.cn/~lgliu/Courses/DGP_2012_spring-summer/Courseware/DGP03_DT-and-MeshGeneration.pdf
+- Surazhsky and Gotsman, Explicit Surface Remeshing, 2003, http://mesh.brown.edu/DGP/pdfs/Surazhsky-sgp03.pdf
```
Comment (changkun) [inline projects/alvaxgln/proposal.md:53]: The link is not accessible.

Comment (changkun) [reply projects/alvaxgln/proposal.md:53]: http://www.cs.technion.ac.il/~gotsman/AmendedPubl/SGP/SGP03.pdf

File: projects/alvaxgln/proposal.md (hunk 1 above)
Comment (changkun) [reply projects/alvaxgln/proposal.md:39]: I think we don't have to consider non-manifold meshes in this case and one could simply assume manifold mesh input to simplify the implementation complexity.

**Review by changkun — APPROVED** [review]
Implementing a local regularizer seems to be a quite possible idea and the necessary local operators such as edge flip and minimization of eq (5) of the original paper provide the necessary complexity for an individual project. Hence I would argue to accept this proposal.

An additional note is that it is necessary to make a clear assumption in order to simplify the handling complexity. As I think this will be much clear after the intermediate submission due to the implementation thinking, it would be necessary to clarify this on the project readme.

**Pull request merged**